documentation/csv-schemas.md

```diff
@@ -54,6 +54,7 @@ Note: Cards in are organized by what main set they were initially released in in
 | Set ID | string | The set identifier for this card printing. | WTR |
 | Edition | string | The edition for this card printing. | U |
 | Rarity | string | The rarity for this card printing. | C |
+| Expansion Slot | string | Whether the card is an expansion slot card or not. | Yes |
```

Comment:
### ⚠️ Suggestion, blocking

I have been documenting "Yes/No" string fields as `bool`, since that's essentially what they are being used as

Reply:
Fixed.